phys/module_cu_gf_wrfdrv.F

```diff
@@ -218,8 +254,12 @@ SUBROUTINE GFDRV(spp_conv,pattern_spp_conv,field_conv,       &
 
      integer, dimension (its:ite) :: cactiv
      real,    dimension (its:ite,kts:kte) ::  qcheck
-
-
+!++ GF CTRAN ++ lyy 01/2020
```

Comment:
Initially there were a couple of blank lines to separate the declarations from the executable code. Do not change change code for no reason.